File: spec/lua/test_default_endian_expr_exception.lua
local luaunit = require("luaunit")

require("default_endian_expr_exception")

TestDefaultEndianExprException = {}

function TestDefaultEndianExprException:test_default_endian_expr_exception()
    luaunit.assertError(DefaultEndianExprException.from_file, "src/endian_expr.bin")
end

File: spec/python/test_default_endian_expr_exception.py
import unittest

from default_endian_expr_exception import DefaultEndianExprException
import kaitaistruct

class TestDefaultEndianExprException(unittest.TestCase):
    def test_default_endian_expr_exception(self):
        with self.assertRaises(Exception):
            with DefaultEndianExprException.from_file("src/endian_expr.bin"):
                pass
